app/vue.config.js

```diff
@@ -1,11 +0,0 @@
-const WorkerPlugin = require('worker-plugin');
-module.exports = {
-  configureWebpack: {
-    output: {
-      globalObject: 'this',
-      hashFunction: 'sha256', // avoids error ERR_OSSL_EVP_UNSUPPORTED
-    },
-    plugins: [new WorkerPlugin()],
```

Comment:
without this the plugin will not be accessible in the demo app; maybe this is why the wfs/wms components don't work anymore?